### ISSUE TITLE
Pytorch 1.2.0 compatibility

### ISSUE THREAD

**Issue by weihua916** [rusty1s/pytorch_geometric#607]
Thanks for the great package!
When I upgraded Pytorch to ver 1.2.0 (just released today), I had the following error. Could you handle this problem (maybe in the next update)? Thanks.

```
File "/anaconda3/lib/python3.7/site-packages/torch_geometric/__init__.py", line 2, in <module>
    import torch_geometric.nn
  File "/anaconda3/lib/python3.7/site-packages/torch_geometric/nn/__init__.py", line 2, in <module>
    from .data_parallel import DataParallel
  File "/anaconda3/lib/python3.7/site-packages/torch_geometric/nn/data_parallel.py", line 5, in <module>
    from torch_geometric.data import Batch
  File "/anaconda3/lib/python3.7/site-packages/torch_geometric/data/__init__.py", line 1, in <module>
    from .data import Data
  File "/anaconda3/lib/python3.7/site-packages/torch_geometric/data/data.py", line 7, in <module>
    from torch_sparse import coalesce
  File "/anaconda3/lib/python3.7/site-packages/torch_sparse/__init__.py", line 2, in <module>
    from .coalesce import coalesce
  File "/anaconda3/lib/python3.7/site-packages/torch_sparse/coalesce.py", line 2, in <module>
    import torch_scatter
  File "/anaconda3/lib/python3.7/site-packages/torch_scatter/__init__.py", line 3, in <module>
    from .mul import scatter_mul
  File "/anaconda3/lib/python3.7/site-packages/torch_scatter/mul.py", line 3, in <module>
    from torch_scatter.utils.ext import get_func
  File "/anaconda3/lib/python3.7/site-packages/torch_scatter/utils/ext.py", line 2, in <module>
    import torch_scatter.scatter_cpu
ImportError: dlopen(/anaconda3/lib/python3.7/site-packages/torch_scatter/scatter_cpu.cpython-37m-darwin.so, 2): Symbol not found: __ZN2at19NonVariableTypeMode10is_enabledEv
  Referenced from: /anaconda3/lib/python3.7/site-packages/torch_scatter/scatter_cpu.cpython-37m-darwin.so
  Expected in: flat namespace
 in /anaconda3/lib/python3.7/site-packages/torch_scatter/scatter_cpu.cpython-37m-darwin.so ```

**Comment by WMF1997** [comment]
hello @weihua916 :
just re-install (uninstall and then install again) them

take torch-scatter for example. (in fact, all 5(torch-scatter, torch-sparse torch-cluster torch-spline-conv torch-geometric) should meet the same problem.)

same problem
![2019-08-09 08-35-02 的屏幕截图](https://user-images.githubusercontent.com/37621591/62746180-ffb2d100-ba80-11e9-9c2b-a82fc7f516ab.png)

re-install works fine
![2019-08-09 08-35-09 的屏幕截图](https://user-images.githubusercontent.com/37621591/62746181-00e3fe00-ba81-11e9-8b9f-72ed679ed4bf.png)

PyG:
![2019-08-09 08-44-11 的屏幕截图](https://user-images.githubusercontent.com/37621591/62746398-e5c5be00-ba81-11e9-893c-bc567c59cafe.png)

P.S. I guess, once your pytorch is upgraded (like this issue), the installed extensions have to be re-installed? 
See [FAQ](https://pytorch-geometric.readthedocs.io/en/latest/notes/installation.html#id1)'s 8.

**Comment by rusty1s** [comment]
Perfect answer :) Does this resolve your issues @weihua916?

**Comment by rusty1s** [comment]
The Travis builds now work again with PyTorch 1.2. There was a small incompatibility with `torch-cluster`'s `knn` method, which is fixed in `torch-cluster==1.4.4`. I consider this issue to be solved. Thanks all.
